telosnetwork/telos-wallet
# Feature/directed login native zero

# Fixes #672 #674 

## Description
See Issues

## Test scenarios

1.  Go to [home page](https://deploy-preview-675--wallet-develop-mainnet.netlify.app/) w/out login query param, see default evm login options
2.  See 'Native' updated to 'Telos Zero'
3.  Click 'Telos Zero', see options visible and route query param updated (?login=zero)
4.  Click 'Telos EVM', see options visible and route query param updated (?login=evm) 
5. input direct url including login query param for both zero and evm and see respective login options shown
6. navigate throughout the native side of the wallet, see all routes behaving consistent w/current production with 'native' route changed to 'zero', i.e. '/zero/profile'

## Checklist:
